subsys/nfc/CMakeLists.txt

```diff
@@ -14,3 +14,6 @@ if (CONFIG_NFC_T4T_ISODEP OR
     CONFIG_NFC_T4T_APDU)
 	add_subdirectory(t4t)
 endif()
+
+zephyr_library()
+zephyr_library_sources(nfc_platform_zephyr.c)
```

Comment:
is it enough to have it compiled when CONFIG_NRFXLIB_NFC is set?

Reply:
I briefly looked at the Cmake dependencies and it seems to me that this file might be picked up in samples/projects that use other NFC modules than nrfxlib (NFC poller samples)?

Reply:
Changed. This configuration depends on `CONFIG_NRFXLIB_NFC` now